ml_dtypes/_finfo.py

```diff
@@ -699,6 +696,9 @@ def float_to_str(f):
       _float8_e8m0fnu_dtype: _float8_e8m0fnu_finfo,
   }
   _finfo_name_map = {t.name: t for t in _finfo_type_map}
+  _finfo_cache = {
+      t: init_fn.__func__() for t, init_fn in _finfo_type_map.items()
```

Comment:
Can you add the `# pytype ...` comment from below? `pytype` doesn't know about `__func__`.